vllm/transformers_utils/s3_utils.py

```diff
@@ -46,6 +46,7 @@ def glob(s3=None,
     """
     if s3 is None:
         s3 = boto3.client("s3")
+    path = path + "/" if not path.endswith("/") else path
```

Comment:
```suggestion
    if not path.endswith("/"):
        path = path + "/"
```

I think this is a bit more readable

Reply:
done